src/main/resources/.gitignore

```diff
@@ -1 +1,2 @@
 config.properties
+config.yml
```

Comment:
Usually do that for config directories to have everything about configs at one place and not in the rather long main `.gitignore`.